addons/web_editor/controllers/main.py

```diff
@@ -381,6 +381,8 @@ def save_less(self, url, bundle_xmlid, content):
             # Create a view to extend the template which adds the original file to link the new modified version instead
             IrUiView = request.env["ir.ui.view"]
             view_to_xpath = IrUiView.get_related_views(bundle_xmlid, bundles=True).filtered(lambda v: v.arch.find(url) >= 0)
+            if len(view_to_xpath) > 1:
+                 view_to_xpath.filtered(lambda v: not v.inherit_id)
```

Comment:
@Julien00859 @nim-odoo 
I do not want to be mean... but come on :disappointed: 

1) The two lines diff is not properly indented.
2) The two lines diff does not nothing, so this code was not even tried. `filtered` does not modify the recordset in place...
3) The fix's idea is not correct... the problem here is that there is an xpath on the variables.less file in web_enterprise, so the above condition `v.arch.find(url)` will find that view. Ok filtering on `v.inherit_id` will solve that really specific problem... but if variables.less was added by an inheriting view originally (which is the case for **most** less files), your code will not find **any** view to xpath. The correct fix is probably changing the `v.arch.find(url)` code.

Reply:
1+2) happens when you don't retry the changes you made... not very professional sorry